apps/api/src/app/widgets/widgets.controller.ts

```diff
@@ -476,6 +476,7 @@ export class WidgetsController {
   ): Promise<LogUsageResponseDto> {
     this.analyticsService.track(body.name, subscriberSession._organizationId, {
```

Comment:
suggestion : `subscriberSession._organizationId` should be changed with `subscriberSession._userId` as per `track` method params

Reply:
Nice catch